src/main/java/io/quarkus/gizmo/BytecodeCreator.java

```diff
@@ -946,6 +947,14 @@ default void continueScope() {
      */
     void breakScope(BytecodeCreator scope);
 
+    /**
+     * Go the the top of the given scope. Unlike {@link #continueScope(BytecodeCreator)} this method does not verify if this
+     * bytecode creator is scoped within the given bytecode creator.
+     * 
+     * @param scope
+     */
+    void jumpTo(BytecodeCreator scope);
```

Comment:
This is tricky because by breaking scoping, we get rid of an easy way to reduce our local variable usage (by limiting the lifetime of a value to its containing scope). There are other ways to do it if we decide to later, but they involve liveness tracking which is not very simple.

Reply:
Ok. We don't need to make it part of the public API but add a package-private method on `BytecoreCreatorImpl` instead... and use it only in special cases like this. WDYM?